Documentation/DE/Referenz/Hooks/Index.rst

```diff
@@ -16,115 +16,330 @@
+         *
+         * This function will be called for "my seminars" , "my vip seminars",
+         * "my entered events" lists only.
+         *
+         * @param \Tx_Seminars_FrontEnd_DefaultController $controller the calling controller
+         *
+         * @return void
+         */
+        public function modifyMyEventsListRow(\Tx_Seminars_FrontEnd_DefaultController $controller)
+        {
+            // Your code here
+        }
 
-   class Tx_Seminarspaypal_Hooks_ListView implements Tx_Seminars_Interface_Hook_EventListView {
         /**
-         * Adds a countdown column.
-       *
-       * @param Tx_Seminars_Model_Event $event
-       *        the affected registration
-        * @param Tx_Oelib_Template $template
-        *        the template from which the list row is built
-        */
-         public function modifyListRow(
-                 Tx_Seminars_Model_Event $event, Tx_Oelib_Template $template
-         ) {…}
+         * Modifies the list view footer in a seminars list.
+         *
+         * This function will be called for all types of seminar lists ("topics",
+         * "seminars", "my seminars", "my vip seminars", "my entered events",
+         * "events next day", "other dates").
+         *
+         * @param \Tx_Seminars_FrontEnd_DefaultController $controller the calling controller
+         *
+         * @return void
+         */
+        public function modifyListFooter(\Tx_Seminars_FrontEnd_DefaultController $controller)
+        {
+            // Your code here
+        }
+    }
+
+.. _registrationform_de:
+
+Hooks zum Registrierungsformular
+""""""""""""""""""""""""""""""""
+
+Es gibt 3 Hooks in das Registrierungsformular:
+
+* Bevor der Formularkopf in HTML umgewandelt wird
+* Bevor das Formular selbst in HTML umgewandelt wird
+* Bevor der Formularfuß in HTML umgewandelt wird
+
+Im Formularkopf und -fuß können Sie eigene Marker ausfüllen oder vorhandene Marker-Werte ändern. Zu
+verfügbaren Eigenschaften und Methoden dafür siehe :file:`Classes/Frontend/DefaultController.php`.
+
+Das Registrierungsformular wird von einer eigenen PHP-Klasse erstellt: :file:`Classes/Frontend/RegistrationForm.php`.
+Es bearbeitet sowohl Registrierung alsauch Abmeldung auf einer oder 2 Formularseiten gemäß der Konfiguration.
+Je nachdem, auf welcher Seite man sich befindet bzw. ob es sich um eine Abmeldung handelt, sind bereits Formularwerte
+eingetragen oder nicht. Wenn Sie eigene Formularfelder hinzufügen wollen, müssen Sie die Speicherung in die DB und das
+Ausfüllen passend zu Seite und Status innerhalb des Registrierungs- bzw. Abmeldeprozesses behandeln. Eine Validierung
+erfolgt bei Bedarf durch `mkforms` (nicht über diese Hooks).
+
+Ihre Klasse, die :php:`\OliverKlee\Seminars\Hooks\Interfaces\SeminarRegistrationForm` implementiert,
+machen Sie seminars in :file:`ext_localconf.php` Ihrer Extension bekannt:
+
+.. code-block:: php
+
+    use \OliverKlee\Seminars\Hooks\Interfaces\SeminarRegistrationForm;
```

Comment:
For the hook registration (which happens in ext_localconf.php etc.), we cannot use classes AFAIK, and need to use FQCN.